config.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"github.com/ddollar/forego/Godeps/_workspace/src/github.com/subosito/gotenv"
```

Comment:
this is a new dependency, but you haven't committed it to this repo or updated Godeps at all. You'd need to `godep save -r ./...` and commit everything new in `./Godeps`